dev.c

```diff
@@ -232,9 +239,10 @@ void fuse_request_send_nowait(struct fuse_conn *fc, struct fuse_req *req)
 	}
 }
 
-static int request_pending(struct fuse_conn *fc)
+static bool request_pending(struct fuse_conn *fc)
 {
-	return fc->queue->r.read != fc->queue->r.write;
+	struct fuse_queue_cb *cb = &fc->queue->requests_cb;
+	return cb->r.read != cb->r.write;
```

Comment:
this looks like the call for px-storage to check whether requests are pending unprocessed. So "r" is owned and moved by px-storage?

Reply:
yes for requests queue